src/parser/jobs/sch/modules/index.js

```diff
@@ -1,18 +1,18 @@
-import ChainStrat from './ChainStrat'
 import DoTs from './DoTs'
 import Interrupts from './Interrupts'
 import Overheal from './Overheal'
 import Pets from './Pets'
+import ScholarCooldownDowntime from './ScholarCooldownDowntime'
```

Comment:
```suggestion
import {CooldownDowntime} from './ScholarCooldownDowntime'
```

As an aside, these imports are somewhat against convention and should be updated, not sure how this even made it into the endwalker branch with default exports.